backend/API/API/HostedServices/IsSolvedService.cs

```diff
@@ -9,19 +13,36 @@ namespace API.HostedServices
     public class IsSolvedService : BackgroundService
     {
         IHubContext<Hubs.OrToolsHub> hubContext;
+        IConsumer consumer;
 
-        public IsSolvedService(IHubContext<Hubs.OrToolsHub> hubContext)
+        public IsSolvedService(IHubContext<Hubs.OrToolsHub> hubContext, IConnectionProvider factory, IConfiguration configuration)
         {
+            QueueService.Models.Settings settings = new QueueService.Models.Settings();
+            configuration.Bind("RabbitMq:IsSolved", settings);
+
+            this.consumer = factory.Connect(settings);
+
             this.hubContext = hubContext;
         }
-        
+
+        public override void Dispose()
+        {
+            base.Dispose();
+            consumer.Dispose();
+        }
 
         protected async override Task ExecuteAsync(CancellationToken stoppingToken)
         {
             while (!stoppingToken.IsCancellationRequested)
             {
-                await Task.Delay(5000);
-                await hubContext.Clients.All.SendAsync("Solved", true);
+                QueueService.Models.ReceiveData receiveData = await Task.Run(() => consumer.Receive(2500));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4618369)